Source/Core/AudioCommon/AlsaSoundStream.cpp

```diff
@@ -19,6 +19,9 @@ AlsaSound::~AlsaSound()
 {
   m_thread_status.store(ALSAThreadStatus::STOPPING);
 
+  // Immediately lock and unlock mutex to prevent cv race.
+  std::unique_lock<std::mutex>{cv_m};
```

Comment:
Where is the lock being released here? Unless I'm missing something, the waiting thread would never get past the cv wait, since it can't re-acquire the mutex, and with the join() would deadlock.

Reply:
@stenzek This creates an object with no name that is immediately destructed before the next line.
I can change it to `{ std::unique_lock<std::mutex> lk{cv_m}; }` if you'd like.

Reply:
Ah, whoops, I didn't see it was a temporary. That's fine then :)